cmake/legacy.cmake

```diff
@@ -151,7 +151,7 @@ if(ICU_ROOT)
   set(dds_icu_hint "-DDDS_LD_LIBRARY_PATH=${ICU_ROOT}/lib")
 endif()
 list(APPEND packages dds)
-set(dds_version "3.6")
+set(dds_version "3.7.23")
```

Comment:
1. Can you please also update `legacy/README.md` with the new version?
2. I don't know, if the status has changed, but once we were told, that DDS should ship "stable versions". So this has changed?

Reply:
1. Sure!
2. Don't know, but fine with me if you wanna do that rule. The version which is in dev now didn't compile, so I took the latest. But also I remember @AnarManafov said he will publish stable one this month?

Reply:
Yes, he will do within the next week or so. Starting with the next stable release he will change DDS`  release numbering so every every release is considered stable (no distinction any more between even and uneven numbers).

Reply:
> 1. Sure!
> 2. Don't know, but fine with me if you wanna do that rule. The version which is in dev now didn't compile, so I took the latest. But also I remember @AnarManafov said he will publish stable one this month?

Hey @rbx ,
yes, sure thing. I am about to release 3.8 either this week or max beg. next. 
I have finished all tests for OS X (intel/arm). Now finishing up the linux part. Once all green, I will push the release.

Reply:
@AnarManafov Current `master` branch of DDS **requires** CMake 3.23. FairSoft currently only requires CMake 3.19. Bumping this because of DDS basically means we have to bootstrap a custom CMake on practically every linux distro in our support matrix. Is DDS really using any CMake 3.23 features? Would it be possible to relax this to 3.19?

Reply:
@dennisklein , honestly I don't recall why the cmake version was bumped. I can for sure relax the requirement.
If you have any machine with 3.19, could you try to just cmake DDS, to make sure nothing comes up?

Reply:
> If you have any machine with 3.19, could you try to just cmake DDS, to make sure nothing comes up?

Not really, but you may download older pre-compiled CMake versions e.g. here https://cmake.org/files/v3.19/. Just unpack it to a custom directory and prepend this `<custom-dir>/bin` to your `$PATH` temporarily, it will not mess with your system CMake.

Reply:
you probably want a minimum of 3.19.5 because of Boost 1.75, see https://gitlab.kitware.com/cmake/cmake/-/commit/82930647607d23b118c84194afc3c70d6d8e79e3.

Reply:
@dennisklein , sounds good. Will do. If works, would 3.19.5 be ok for you as a minimum?

Reply:
@dennisklein , thank you. I have checked 3.19.5. Everything looks ok. As you requested, I relaxed the min cmake version to 3.19. the change is in the master. Once I finish all linx tests, I will push for a tagged release.